episodes/09-eia.md

```diff
@@ -55,7 +55,7 @@ things that we do, things that we are, and things that we champion. Many of thes
 
 ## Discuss The Carpentries Core Values
 
-1. Take a moment to read through the Core Values on this page: [https://carpentries.org/values/][core-values]
+1. Take a moment to read through the [Core Values][core-values]
```

Comment:
i think this is like this in order to help the etherpad transfer? @karenword

Reply:
yes thanks for the catch @brownsarahm -- all links in callouts are written out in the text to make them easier to copy/paste into the Etherpad.